apps/antalmanac/src/components/Header/Import.tsx

```diff
@@ -66,7 +69,7 @@ function Import() {
         const sectionCodes = zotcourseImport ? zotcourseImport.codes : studyListText.match(/\d{5}/g);
 
         if (!sectionCodes) {
-            openSnackbar('error', 'Cannot import an empty/invalid Study List/Zotcourse.');
+            openSnackbar('error', `Cannot import an ${studyListText ? 'invalid' : 'empty'} Study List.`);
```

Comment:
I know you didn't write this, but we should also use `importSource` over the existence of `zotcourseImport` to determine which value to assign

```
        const importZotcourse = importSource === 'zotcourse'
        // ...
        const sectionCodes = importZotcourse ? zotcourseImport?.codes : studyListText.match(/\d{5}/g);
        
        if (!sectionCodes) {
            openSnackbar('error', `Cannot import an empty ${importZotcourse ? 'Zotcourse' : 'Study List'.`);
```